### DISCUSSION TITLE
there is no documentation for loacal setup, i dont know what to do

### DISCUSSION THREAD

**Discussion by varada13** [mirumee/saleor-dashboard#3665] [Q&A]
i have created env file and added variables, i dont know what to add in APPS_MARKETPLACE_API_URI.
if i  run cp .env.template .env in terminal, it is showing api is readonly and cant do any thing ,
why is there no guide to setup the project ? how do people set up this?

can anyone please explain to me any of this?
for 1 week i have been trying and not found any solution.

**Comment by chrislaai** [comment]


**Reply by varada13** [reply]
how to set up that Saleor API?
any examples ?
i have no idea how to set up the API
all i did was cloned and run the core, storefront and dashboard repositories.
please help

**Reply by parmanand007** [reply]
localhost/:1 Not allowed to load local resource: file:///D:/code/frontend/saleor-dashboard/src/index.tsx

i am facing this issue in console of dashboard .. what to do?

**Reply by rino92xxl** [reply]
Also getting this issue: Loading failed for the module with source “c:/code/saleor-dashboard/src/index.tsx”.

**Reply by EL-MEHDI-ESSAADI** [reply]
making the entry level equal **"/index.tsx"** solved "Not allowed to load local..." problem for me.
![image](https://github.com/saleor/saleor-dashboard/assets/79756353/513f55d0-6f11-49b9-9f5d-1c5449c5fd7d)
